src/app/game/play/player-card/QuestionBody.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { useCustomTranslation } from '../../../hooks';
+import { Styled } from './styled';
+
+export const QuestionBody = () => {
+  const { t } = useCustomTranslation();
+
+  return (
+    <>
+      <Styled.PlayerName>{t('game.you')}</Styled.PlayerName>
```

Comment:
I'd extract these hardcoded strings ('game.you') in to a separate file and make them constants